### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by Jeanbulambo**
## details popup window
# in this project we implement with Daniel the following items:
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- You should implement popups for both mobile and desktop screen sizes:
- Mobile popup

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,8 +1,192 @@
...
+  btnseesource.classList.add('btn-seesource-pp');
+  btnseesource.target = '_blank';
+  btnseesource.textContent = 'See Source';
+
+  const clickCloseButton = document.getElementById('close-button');
+  clickCloseButton.addEventListener('click', 'closeButtonFunction');
+  // eslint-disable-next-line no-unused-vars
+  function closeButtonFunction() {
+    popup.remove();
+  }
```
Comment (RolandM99) [inline script.js]: - [ ] Kindly, make sure your close button work as expected. For now, when we click on the close button (X) nothing happens, the popup window doesn't disappear.

File: style.css [hunk 2]
```diff
@@ -869,6 +873,62 @@ ul {
...
+  .header-pp {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  .pop-up-class {
+    width: fit-content;
+    height: max-content;
+    flex-wrap: wrap;
```
Comment (RolandM99) [inline style.css]: - [ ] As asked on the project requirement, your design should look exactly like the design from Figma. Kindly, make sure that your pop-up card for the mobile version fit the popup designed on the Figma framework. The margin, padding, and width,... should be the same.

|Your Design|Figma Design|
|-------------|---------------|
|image|image|

File: style.css [hunk 3]
```diff
@@ -869,6 +873,62 @@ ul {
...
+    display: block;
+  }
+
+  .mbtn {
+    display: none;
+  }
+
+  .position-div-pp {
+    display: flex;
+    align-items: baseline;
```
Comment (RolandM99) [inline style.css]: - [ ] Make sure the elements underlined here below has the same margin-top as margin-left characteristics like on the Figma design. Kindly, fix this issue:
image

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @Jeanbulambo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -442,9 +443,9 @@ ul {
   display: none;
 }
 
-/* pop up menu classes */
+/* pop up window part for mobile version */
 
-.header-pp {
+.headpop {
```
Comment (Kossi-stack) [inline style.css:448]: - [ ] Kindly fix the popup for the mobile version the (x) button is not working and is not well style.
Captureusainm

File: style.css [hunk 5]
```diff
@@ -459,7 +460,7 @@ ul {
   float: right;
 }
 
-.pop-up-class {
+.popupc {
```
Comment (Kossi-stack) [inline style.css:467]: - [ ] Kindly fix the popup for the desktop version is not well style.
Captureusain

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Jeanbulambo,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 6]
```diff
@@ -894,14 +894,14 @@ ul {
...
+    height: 821px;
     flex-wrap: wrap;
     flex-direction: column;
-    justify-content: center;
+    /* justify-content: center; */
     align-items: center;
-    margin: 1px 14px;
-    left: 1px;
+    left: 45px;
+    margin: 0 145px;
```
Comment (Kandy-Peter) [inline style.css]: - [ ] You still have the same issues with your `popup` window. I think this is due to the element alignment and elements `position`. Please ensure that your `nav` menu is positioned relative to its parent container i.e if the button has as position `absolute` the parent's container should have `position: relative`.
- [ ] Also, make sure that the design desktop design is similar to the Figma template design.  In your case the alignment is too dissimilar, please take the time to resolve it. I know you can do it so go ahead and have good courage! 😊 

Your design

image

The Figma design

image

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @Jeanbulambo,

Good job so far! I know you have worked hard on implementing the changes requested by the previous reviewer. However, there is an issue that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Jeanbulambo 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

- [ ] [optional] the problem that requested you the previous reviewer is still not solved but it's one of the requirements even though you should fix that 
image
- [ ] [optional] please make sure that  the button of the pop-up have the same font and the effect on hover and pressed as the ones in the Figma design 
image


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**